frontend/packages/component-library/src/accordion/faqAccordion/stories/FAQAccordion.story.tsx

```diff
@@ -91,3 +91,94 @@ export const WithLongAnswers: Story = {
     ],
   },
 };
+
+export const WithMultipleParagraphsAsContent: Story = {
```

Comment:
Thanks for adding this! 🙌 

This doesn't work in Contentful, is this something we want to support now or add later cc @moshebaricdo? The screenshot below is showing a paragraph break in the [entry](https://app.contentful.com/spaces/90t6bu6vlf76/environments/master/entries/1KksgPlhLWJax7lRv98F5B?focusedField=question&focusedLocale=en-US) but it doesn't render that way:

<img width="1014" alt="Screenshot 2025-03-11 at 2 28 25 PM" src="https://github.com/user-attachments/assets/9b0976cf-0048-4cd7-904c-bcd8eef85359" />

Reply:
Good question. Reason of the diff is that in contentful we have a plain text prop (long string) and in storybook there's a React Node with multiple p elements provided.

In order to support multiparagraphs in Contentful, as well as any other ReactNode related functionality - we will need to implement Rich Text support.

Reply:
Ok cool! When Rich Text is supported do you think we should default to that and remove the plain text support/content type?

Reply:
Not sure, we can come back to this once we have the rich text support and make a decision.
Currently there are two different content types - faq with plain text and faq with rich text(though it's not supported now), by the time we have rich text we should have some feedback on this and be able to make a decision